test/get-workspaces-paths-test.js

```diff
@@ -221,4 +224,208 @@ describe(function() {
       });
     });
   });
+
+  describe('spawn with yarn', function() {
+    describe('workspaces array', function() {
+      beforeEach(function() {
```

Comment:
Let's reuse the beforeEach code from above. And make new `it` blocks instead, like I did with sync/async.